R/tt_paginate.R

```diff
@@ -205,7 +205,7 @@ setMethod(
       path <- c(path, obj_name(tt))
     }
     ret <- list()
-    
```

Comment:
also these, we should have activated the linter checks at the end, right?

Reply:
there is no lintr in here 🤷